src/custom-elements/src/components/app-layout/app-layout.tsx

```diff
@@ -81,6 +60,7 @@ export class AppLayout {
 
     if (this.el.getAttribute('title')) {
       this.appLayoutTitle = this.el.getAttribute('title');
+      this.el.removeAttribute('title');
```

Comment:
Please, add this action to the connect event as well, just in the case the component is reused.